common/djangoapps/philu_commands/management/commands/auto_generate_certificates_for_open_courses.py

```diff
@@ -69,18 +69,21 @@ def handle(self, *args, **options):
         for course in modulestore().get_courses():
             if not is_course_valid_for_certificate_auto_generation(course):
                 continue
-
+            log.info("course id : {course_id}".format(course_id=course.id))
             for user_course_enrollment in CourseEnrollment.objects.filter(course_id=course.id, is_active=True).all():
                 user = user_course_enrollment.user
                 cert_data = _get_cert_data(user, course, user_course_enrollment.mode)
                 if not cert_data or cert_data.cert_status != CertificateStatuses.requesting:
+                    log.info("skipping because status is : {cert_status}".format(cert_status=cert_data.cert_status))
                     continue
                 course_chapters = modulestore().get_items(
                     course.id,
                     qualifiers={'category': 'course'}
                 )
 
                 if _is_eligible_for_certificate(user_course_enrollment, course_chapters, user):
+                    log.info("skipping because course with id {course_id} is not eligible for certificate".format(
+                        course_id=course.id))
```

Comment:
Please also reformat to 
```
                   log.info("skipping because course with id {course_id} is not eligible for certificate".format(
                        course_id=course.id
                   ))
```

Reply:
@zeshanarifios please review.